apps/web/app/api/v1/utils/types.ts

```diff
@@ -7,11 +7,17 @@ export const zStringBool = z
   .refine((val) => val === "true" || val === "false", "Must be true or false")
   .transform((val) => val === "true");
 
-export const zGetBookmarkSearchParamsSchema = z.object({
+export const zGetBookmarkQueryParamsSchema = z.object({
```

Comment:
`zGetBookmarkQueryParamsSchema`: not used in /bookmarks/search
`zGetBookmarkSearchParamsSchema `: used in /bookmarks/search